pyaedt/modules/PostProcessor.py

```diff
@@ -4055,7 +4055,7 @@ def export_model_obj(self, assignment=None, export_path=None, export_as_single_o
             return [[fname, "aquamarine", 0.3]]
 
     @pyaedt_function_handler(setup_name="setup")
-    def export_mesh_obj(self, setup=None, intrinsics=None):
+    def export_mesh_obj(self, setup=None, intrinsics=None, no_vacuum=True, on_surfaces=True):
```

Comment:
```suggestion
    def export_mesh_obj(self, setup=None, intrinsics=None, air_objects=True, on_surfaces=True):
```

To keep consistency with plot method.